apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -57,14 +57,15 @@ Page {
                 EnterKey.onClicked: focus = false
             }
 
-            ComboBox {
+            PermissionListItem {
                 id: searchEngine
                 enabled: AccessPolicy.browserEnabled
                 width: parent.width
                 //: Label for combobox that sets search engine used in browser
-                //% "Search with"
+                //% "Search engine"
```

Comment:
Please do not bring back the word "engine".

Reply:
@rainemak Also on design